Sources/Extensions/AppKit/NSViewExtensions.swift

```diff
@@ -132,7 +132,7 @@ extension NSView {
     ///
     /// - Parameter subviews: array of subviews to add to self.
     public func addSubviews(_ subviews: [NSView]) {
-        subviews.forEach({self.addSubview($0)})
+        subviews.forEach({addSubview($0)})
```

Comment:
I know is not the purpose of the PR but can you also fix here to use trailing syntax and adding space between like `subviews.forEach { addSubview($0) }`? :))